src/lwc/rd2EntryFormDonorSection/rd2EntryFormDonorSection.js

```diff
@@ -86,6 +86,7 @@ export default class rd2EntryFormDonorSection extends LightningElement {
         } else if (parentSObjType === CONTACT_OBJECT.objectApiName) {
             this.contactId = this.parentId;
             this.donorType = 'Contact';
+            this.dispatchContactChangeEvent(this.contactId);
```

Comment:
@rponti-sforg , the new Recurring Donation test cases should cover creating it from a:
- Contact detail page where MailingCountry = US
- Contact detail page where MailingCountry = CA
- Contact detail page where MailingCountry is empty
- Account detail page
- Recurring Donations tab

Reply:
Moreover, it should be verified that Recurring Donations on an org not connected to Elevate continue to be working as before, ie the functionality is not broken.